src/main/java/duke/Command.java

```diff
@@ -0,0 +1,258 @@
+package duke;
+
+import java.util.ArrayList;
+
+/**
+ * Deals with the Command given by user and implement the corresponding actions.
+ */
+public abstract class Command {
+    /**
+     * Abstract method meant to indicate the actions taken for different command.
```

Comment:
> In method header comments, the first sentence should start in the form Returns ..., Sends ..., Adds ... (not Return or Returnning etc.)

Would it be better this way?

```suggestion
     * Indicates the actions taken for different command.
```

A client who uses this class does not need to know that the method is abstract. A subclass of `Command` would know it is abstract, otherwise the code will not compile.

[Here](https://docs.oracle.com/en/java/javase/11/docs/api/java.desktop/javax/swing/Action.html) is an example from Java 11 API. Notice that `getValue(String key)` is listed under abstract methods, but its Javadoc does not mention anything about it being `abstract`.

Reply:
Many thanks for all your comments, I'll change it accordingly.